a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -203,6 +205,21 @@ def get_delete_load_balancer_flow(self, lb, deleteCompute):
 
         return (delete_LB_flow, store)
 
+    def _get_delete_lb_vlan_network_subflow(self):
```

Comment:
Don't need a subflow for this. This can be compressed into a `DeleteEthernetTagIfNotInUse` task

Reply:
Done.